paseos/actors/spacecraft_actor.py

```diff
@@ -2,18 +2,30 @@
 import pykep as pk
 
 from paseos.actors.base_actor import BaseActor
+from paseos.power import discharge_model
+from paseos.power import charge_model
+from paseos.power.is_in_eclipse import is_in_eclipse
 
 
 class SpacecraftActor(BaseActor):
     """This class models a spacecraft actor which in addition to pos,
     velocity also has additional constraints such as power/battery."""
 
-    # Power constraints
-    # TODO
-    _available_power = None
+    # Power-related properties
+    battery_level_in_Ws = None
```

Comment:
I think most people know Watt and with the combination of wattseconds it is very intuitive. J is less well known especially for non-electrical-engineering folks, so it might be more understandable this way?

Reply:
I think J is pretty common, but let's leave Ws for the moment.